File: FriendstagramClient/src/app/models/sharing.ts
import { AppUser } from './app-user';
export class Sharing{
  public sharingId:number = 0;
  public userId:number = 0;

  public path:string = "";
  public sharingText:string = "";

  public showComment:boolean = false;

  public user?:AppUser = new AppUser();

  public comments?:any = "";

}

File: FriendstagramApi/FriendstagramApi/Entities/Models/Sharing.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace FriendstagramApi.Entities.Models
{
    public class Sharing
    {
        public int SharingId { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public string Path { get; set; }
        public string SharingText { get; set; }
    }
}
